### DISCUSSION TITLE
Dealing with disordered proteins

### DISCUSSION THREAD

**Discussion by B-Bajramovic** [KosinskiLab/AlphaPulldown#166] [Ideas]
Dear developers,

I wish to learn if there are any solid approaches to dealing with protein disorder using AF-multimer. 

For a protein pair, we have experimentally verified that the structured region of A is most important for interaction with B, but models include many additional contact pairs in the long disordered tail of A. This negatively influences confidence scoring, making the overal model seem unreliable.

To circumvent this, I tried cutting off the disordered tail of A, but this resulted in strange secondary structures. I tried many variations of this, but none seemed to maintain secondary structure of the region of interest.

Without changing input, is there a way to exclusively score the structured interface? Does it even make sense to try this?

best wishes,
Belmin, Leiden University

**Comment by dingquanyu** [comment]
Hi,

Sorry since it is not related to the software issue I will transfer this to a discussion and see if any other users have suggestions on your case.

**Comment by jkosinski** [comment]
Cutting disordered regions out is likely the only way for now and should generally work. How did you cut the tail off? By using the AlphaPulldown interface to specify residue ranges in proteinA,1-200 format?

**Comment by MHvegholm** [comment]
I'm currently facing the same challenges, I suspect the score itself isn't affectable without compromising the reliability of the results. Instead, my best guess on working around it would be to rely on the `create_notebook.py` script to sift out interesting predictions and manually determine the quality of the candidates. I imagine tweaking the `--cutoff` variable of `create_notebook.py` could result in a useful alternative, though it doesn't solve the inital problem.

**Comment by jkosinski** [comment]
The score will be affected but we assume that it is affected for the better. We've seen cases where interactions only showed up after removing extra domains or disordered tails. Chopping proteins into a smaller fragment is a common practice, but probably no one checked systematically how that affects the reliability of scoring. Reassuring is that long proteins are chopped into small fragments even in the official AlphaFold Database.
